packages/tabs-overflow/README.md

```diff
@@ -0,0 +1,45 @@
+## Description
+
+The `<sp-tabs-overflow>` is a decorative component that works in conjunction with the `<sp-tabs>` component. It enables horizontal tab scrolling in cases where the available width is insufficient to display all tabs.
+
+_Note: Veritical scrolling is not yet supported._
+
+### Usage
+
+[![See it on NPM!](https://img.shields.io/npm/v/@spectrum-web-components/tabs-overflow?style=for-the-badge)](https://www.npmjs.com/package/@spectrum-web-components/tabs-overflow)
+[![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/tabs-overflow?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/tabs-overflow)
+
+```
+yarn add @spectrum-web-components/tabs-overflow
+```
+
+To import the `<sp-tabs-overflow>` component with its associated side effects, use the following import statement:
+
+```
+import '@spectrum-web-components/tabs-overflow/sp-tabs-overflow.js';
+```
+
+To utilize the `TabsOverflow` base class as a type or for extending its functionality, utilize the following method:
+
+```
+import { TabsOverflow } from '@spectrum-web-components/tabs-overflow';
+```
+
+## Example
+
+To use the `<sp-tabs-overflow>` component, simply wrap it around the `<sp-tabs>` element as a child element:
+
+```html
+<sp-tabs-overflow>
```

Comment:
We likely want to do something with this demo that will _always_ trigger the overflow UI, maybe a `max-width` or something?